src/Backend/Modules/ContentBlocks/Command/CreateContentBlockHandler.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Backend\Modules\ContentBlocks\Command;
+
+use Backend\Core\Engine\Model;
+use Backend\Modules\ContentBlocks\Entity\ContentBlock;
+use Backend\Modules\ContentBlocks\Event\ContentBlockCreated;
```

Comment:
Should Event class names be added the "Event" postfix?
`ContentBlockCreatedEvent`